CHANGELOG/CHANGELOG-v0.11.md

```diff
@@ -0,0 +1,39 @@
+# v0.11.0
+
+## Changelog since v0.11.0
+
+### New Features
+* Add installStrategy in addon v1alpha1 API ([#193](https://github.com/open-cluster-management-io/api/pull/193) [@qiujian16](https://github.com/qiujian16))
```

Comment:
should this point to https://github.com/open-cluster-management-io/api/pull/207 ?

Reply:
good catch